Mapsui.Rendering.Xaml/ArrowLineStringRenderer.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Mapsui.Geometries;
+using Mapsui.Styles;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using XamlMedia = System.Windows.Media;
+using System.Windows.Shapes;
+using XamlPoint = System.Windows.Point;
+using XamlColors = System.Windows.Media.Colors;
+
+namespace Mapsui.Rendering.Xaml
+{
+    public static class ArrowLineStringRenderer
+    {
+        public static Shape Render(ArrowLineString arrowLineString, IStyle style, IViewport viewport, SymbolCache symbolCache)
+        {
+            if (!(style is VectorStyle)) throw new ArgumentException("Style is not of type VectorStyle");
+            var vectorStyle = style as VectorStyle;
```

Comment:
Oh nice, you did it exactly like I was saying in my comments above on the Skia renderer!